script.js

```diff
@@ -0,0 +1,96 @@
+const courseContainer = document.querySelector(".courses-container");
+const searchButton = document.querySelector(".search-btn");
+let courses = [];
+searchButton.addEventListener("click", function(e) {
+    e.preventDefault();
+    updateTable();
+});
+const searchText = document.querySelector(".search-input");
+searchText.addEventListener("keyup", function(e) {
+    updateTable();
+});
+
+function updateTable() {
+    let here = [];
+    here = filter(searchText.value);
```

Comment:
you can do `const here=filter(searchText.value);` in one line right? also what is `here` used for? could you give it meaningful num?

Reply:
yeah it can be in one line , here means the current courses after filtering , i will rename it okay